src/test/ui/const-generics/issues/auxiliary/const_generic_issues_lib.rs

```diff
@@ -1,4 +1,6 @@
-#![feature(const_generics)]
+#![cfg_attr(full, feature(const_generics))]
+#![cfg_attr(full, allow(incomplete_features))]
+#![cfg_attr(min, feature(min_const_generics))]
```

Comment:
I am not too sure on how `aux-build` works, are these `cfg_attr` used?

I can look into this myself later if you don't have the time to do so.

Reply:
They are, with a quick test. Removing these yields a "const generics are unstable" error in `ui/const-generics/issues/issue-73120.rs`.

That file directly doesn't use const generics but the auxiliary library here does.

Reply:
:+1: